modules/nodes-cluster-resource-levels-command.adoc

```diff
@@ -10,7 +10,7 @@ to estimate the number of pods that can be scheduled onto your cluster.
 
 .Prerequisites
 
-* Download and install link:https://github.com/kubernetes-incubator/cluster-capacity[the *cluster-capacity* tool].
+* Run link:https://catalog.redhat.com/software/containers/openshift4/ose-cluster-capacity/5cca0324d70cc57c44ae8eb6?container-tabs=overview[the *OpenShift Cluster Capacity Tool*] available as a container image from the Red Hat Ecosystem Catalog.
```

Comment:
* I'd put "the" outside of the link text
* The tool name shouldn't be in bold (bold is usually reserved for GUI elements
* Maybe adding a ", which is"
```suggestion
* Run the link:https://catalog.redhat.com/software/containers/openshift4/ose-cluster-capacity/5cca0324d70cc57c44ae8eb6?container-tabs=overview[OpenShift Cluster Capacity Tool], which is available as a container image from the Red Hat Ecosystem Catalog.
```

Reply:
Done